ipn/ipnlocal/local.go

```diff
@@ -3296,7 +3296,7 @@ func (b *LocalBackend) validPopBrowserURL(urlStr string) bool {
 	if err != nil {
 		return false
 	}
-	serverURL := b.Prefs().ControlURLOrDefault()
+	serverURL := b.sanitizedPrefsLocked().ControlURLOrDefault()
```

Comment:
If we expect this function to always be called with `b.mu` held, would not we want to rename it accordingly (`validPopBrowserURLLocked`) and audit/change all callsites, making sure that the lock is held?

Reply:
You're right, I thought this was a method we added. It isn't... So this is not the right approach.